### ISSUE TITLE
No headers are installed with `USE_CXX=off`

### ISSUE THREAD

**Issue by kmarius** [dankamongmen/notcurses#2674]
I am running the commands as in the [PKGBUILD](https://github.com/archlinux/svntogit-community/blob/packages/notcurses/trunk/PKGBUILD) but with `-DUSE_CXX=off` (and without the check, and the ffi, which both fail in this case), resulting in a package without headers.

**Comment by kmarius** [comment]
Looks like this https://github.com/dankamongmen/notcurses/blob/ab8590d7b3a7a3129a603e4235ff50eb2b0cfd9f/CMakeLists.txt#L577 is inside a large `if(${USE_CXX})` block.

**Comment by aghast** [comment]
I had this same problem. I'm utterly incompetent at cmake, so I'm glad someone else reported it. I've been building the entire thing into an intermediate install directory and picking out the C-related parts that I care about as part of my do script. 

I found that there is an `endif()` fairly close to that headers block, which I moved. The result does build, and it puts the C headers in the install dir without having to USE_CXX. I'd like to be able to say the library works, but I haven't been able to make that happen. (That was true even with the CXX stuff being built, so I think it's an unrelated item. I can't even get the blue house demo to work from the book online-- somehow stdin is always closed immediately.)
